features/location/impl/src/main/kotlin/io/element/android/features/location/impl/view/LocationActions.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.features.location.impl.view
+
+import androidx.compose.runtime.Composable
+import io.element.android.features.location.api.Location
+
+interface LocationActions {
+
+    @Composable
+    fun Configure()
```

Comment:
I don't really like this strategy, but I guess it's needed to provide the activity context instead of the application one since we can't open external applications using the app context, right?

Reply:
Yeah 🫤

(This is the same pattern used for local media actions, so if we do come up with a nicer way of doing it we can fix them all at once)

Reply:
About this: since this is a share action, I think we do want to start whatever new activity appears here as a new task, don't we? We could just use the app context and add the missing `FLAG_ACTIVITY_NEW_TASK` in that case. From the docs:

> This flag is generally used by activities that want to present a "launcher" style behavior: they give the user a list of separate things that can be done, which otherwise run completely independently of the activity launching them.

Reply:
Ooh, yes. Good catch, thanks.